include/stdx/ct_string.hpp

```diff
@@ -78,6 +78,18 @@ template <ct_string S, char C> [[nodiscard]] consteval auto split() {
     }
 }
 
+template <std::size_t N, std::size_t M>
+[[nodiscard]] constexpr auto operator+(ct_string<N> const &lhs,
+                                       ct_string<M> const &rhs)
+    -> ct_string<N + M - 1> {
```

Comment:
question: why -1 on the final size?

Reply:
The strings store null terminators. In the concatenation we only want one null terminator.